docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>Change the following:</p><pre class="ql-syntax" spellcheck="false">Avro data is in row-oriented, compacted binary data format. It relies on schema. When Avro data is stored in a file, its schema is stored with it, so that files may be processed later by any program. Definition of schema is represented by JSON.
</pre><p>to:</p><pre class="ql-syntax" spellcheck="false">Avro data is a row-oriented, compacted binary data format. It relies on schema which is stored as a separate JSON file.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
thanks, updated.